### PR TITLE
chore: packages update and min php version 8.1

### PR DESCRIPTION
**Pull request by bradymiller**
chore: packages update and min php version 8.1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "openemr/openemr",
-    "version": "7.0.0",
+    "version": "7.0.2",
```
Comment (stephenwaite) [inline composer.json:3]: cool

File: interface/procedure_tools/gen_universal_hl7/gen_hl7_order.inc.php [hunk 2]
```diff
@@ -532,7 +532,7 @@ function send_hl7_order($ppid, $out)
         }
 
         // Connect to the server and write the file.
-        $sftp = new \phpseclib\Net\SFTP($remote_host);
+        $sftp = new \phpseclib3\Net\SFTP($remote_host);
```
Comment (stephenwaite) [inline interface/procedure_tools/gen_universal_hl7/gen_hl7_order.inc.php:535]: we could add the `php-ssh2` extension and remove this package someday?

Comment (stephenwaite) [reply interface/procedure_tools/gen_universal_hl7/gen_hl7_order.inc.php:535]: actually the php-ssh2 extension doesn't have the handy classes that the package has

Comment (bradymiller) [reply interface/procedure_tools/gen_universal_hl7/gen_hl7_order.inc.php:535]: yep, there are a couple pain points so far with the package updates.
The google/apiclient updated package is requiring going to phpseclib3 which will involve refactoring code in at least 1 place (sha384 public key verify stuff) and also guessing here.
Even worse it the most updated lcobucci/jwt package has decided to remove the `forUnsecuredSigner` stuff which is causing the normal bloating, nausea, diaphoresis :)
And I am guessing there will be more :)

**Comment by stephenwaite** [comment]
nice

**Comment by bradymiller** [comment]
Testing well and bringing in.

"Let the Games Begin" :) : https://www.open-emr.org/wiki/index.php/QA_and_Release_Process#Version_7.0.2


dance-bear

**Comment by sjpadgett** [comment]
You are a pro Brady no doubt

**Pull request merged**